.golangci.yaml

```
@@ -217,6 +233,13 @@ linters:
           - gomodguard
         path: test/
         text: "logrus"
+      - linters:
+          - kubeapilinter
+        path-except: pkg/k8s/apis/cilium\.io/(v2|v2alpha1)/.*types\.go
+      - linters:
+          - kubeapilinter
+        path: pkg/k8s/apis/cilium\.io/(v2|v2alpha1)/.*types\.go
+        text: deepequal-gen
```

@joestringer 

During the rebase and testing process, I noticed that deepequal is also being flagged by KAL’s duplicateMakers check.
Although deepequal is generated by Cilium using deepcopy and is unrelated to the Kubernetes API, KAL does not provide a way to exclude it.
For this reason, I added this item to suppress the error

---

Sounds good, thanks for highlighting.